### PR TITLE
Release 4.7

### PR DESCRIPTION
**Pull request by ramijebara**
4.7 release notes, home page re-organization.

### CODE REVIEW

File: documentation/index.md [hunk 1]
```diff
@@ -8,89 +8,129 @@ sd_hide_title: true
 :alt: Docs Header Image
 :width: 1104
 ```
+#
```
Comment (kyledickau) [inline documentation/index.md:11]: Is this blank heading intentional?

File: documentation/release-notes/topics/version_4.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+:::
+
+### Enhanced Feature: Device Discovery
+
+Discovery for Cisco switches has been enhanced to auto-detect and track power information. If detected, new sensors will be created to track the power data.
+
+Discovery for Rittal CMC III monitoring system has been improved to detect more environmental sensors.
+
+:::{important}
+These enhancements requires the latest Linux version of the Data Collector.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.md:20]: Should be "require" instead of "requires".

File: documentation/index.md [hunk 3]
```diff
@@ -8,89 +8,129 @@ sd_hide_title: true
...
+
+
+{.frontpage-heading1}
+# Asset Management
```
Comment (kyledickau) [inline documentation/index.md:15]: In my opinion these h1 headings are way too big for this purpose. I think the h3 size (1.5em / 24px) would be better.

File: documentation/release-notes/topics/version_4.md (hunk 2 above)
Comment (ramijebara) [reply documentation/release-notes/topics/version_4.md]: fixed

**Review by kyledickau — APPROVED** [review]

File: documentation/index.md (hunk 3 above)
Comment (ramijebara) [reply documentation/index.md:15]: fixed

**Pull request merged**